mypy.ini

```diff
@@ -0,0 +1,50 @@
+[mypy]
+show_error_codes = True
+namespace_packages=True
+plugins=mypy_zope:plugin
+
+[mypy-idna]
+ignore_missing_imports = True
+
+[mypy-nacl]
+ignore_missing_imports = True
+
+[mypy-nacl.encoding]
+ignore_missing_imports = True
+
+[mypy-nacl.exceptions]
+ignore_missing_imports = True
+
+[mypy-nacl.signing]
+ignore_missing_imports = True
+
+[mypy-netaddr]
+ignore_missing_imports = True
+
+[mypy-OpenSSL]
+ignore_missing_imports = True
+
+[mypy-prometheus_client]
+ignore_missing_imports = True
+
+[mypy-phonenumbers]
+ignore_missing_imports = True
+
+[mypy-signedjson]
+ignore_missing_imports = True
+
+[mypy-signedjson.key]
+ignore_missing_imports = True
+
+[mypy-signedjson.sign]
+ignore_missing_imports = True
```

Comment:
You can condense repeated entries like this down to just `[mypy-signedjson.*]`

But actually, Mypy 0.900 was just [released yesterday](https://mypy-lang.blogspot.com/2021/06/mypy-0900-released.html) and it supports [configuration in pyproject.toml](https://mypy.readthedocs.io/en/latest/config_file.html#using-a-pyproject-toml-file)!

Most notably, you can now condense _all_ of these into lines like:

```toml
[[tool.mypy.overrides]]
module = [
    'signedjson',
    'sortedcontainers',
    'unpaddedbase64',
]
ignore_missing_imports = true
```

Let's do that!

Reply:
You got it!